addons/sales_team/static/src/scss/crm_team_views.scss

```diff
@@ -0,0 +1,44 @@
+$o-kanban-large-record-width: 400px;
+$o-kanban-medium-record-width: 350px;
+
+.o_kanban_view.o_crm_team_kanban {
+    .o_kanban_group:not(.o_column_folded) {
+        width: $o-kanban-large-record-width + 2*$o-kanban-group-padding;
+    }
+    .o_kanban_record {
+        width: $o-kanban-large-record-width;
+    }
+}
+
+.o_kanban_view.o_crm_team_member_kanban {
```

Comment:
Small "oopsie" I noticed with the form view ribbon:

![image](https://user-images.githubusercontent.com/41282536/99803438-bbdec080-2b39-11eb-851f-3ef9dd24d450.png)

Reply:
Did you fix this one after all?